platform/src/main/resources/config/anchor-asset-default-values.yaml

```diff
@@ -0,0 +1,155 @@
+#
+# This file contains the configuration for the assets. The values used here are for testing purposes only.
+#
+
+# The list of the assets that the anchor supports.
+#
+# Note: The `id` of each asset is the unique identifier for the asset which is formated as `schema:code:issuer`.
+#    - `schema`: is the schema of the asset. It can be either `stellar` or `iso4217`.
+#    - `code`: is the code of the asset.
+#    - `issuer`: is the issuer of the asset. It is only used for Stellar assets that are not native.
+# Currently, the anchor supports the following asset schemas:
+#   - `stellar`: The Stellar asset.
+#   - `stellar:native`: The native asset (XLM).
```

Comment:
`native` is essentially under `stellar` scheme, let's not make it a separate item here? We can make an excpetion explanation about it  under the `stellar`.

Reply:
SGTM. 

Removed.